test/functional/grade_entry_forms_controller_test.rb

```diff
@@ -497,15 +497,15 @@ class GradeEntryFormsControllerTest < AuthenticatedControllerTest
 
         put_as @admin,
                :update,
-               :id => @grade_entry_form.id,
-               :grade_entry_form => {
-                   :short_identifier => NEW_SHORT_IDENTIFIER,
-                   :description => NEW_DESCRIPTION,
-                   :message => NEW_MESSAGE,
-                   :date => @grade_entry_form.date,
-                   :grade_entry_items_attributes => {'new_1' => @q2,
-                                                     '1' => @q1,
-                                                     'new_2' => @q3}}
+               id: @grade_entry_form.id,
+               grade_entry_form: {
+                                    short_identifier: NEW_SHORT_IDENTIFIER,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.